NewFile.txt

```diff
@@ -0,0 +1,7 @@
+This is a new file.
+
+With some new content.
+
+Seriously...
+
+Very important content.
```

Comment:
Reply.

---
In reply to: [68509838](https://github.com/kivancmuslu/GitHubTest/pull/1#discussion_r68509838) [](ancestors = 68509838)